<ui version="4.0" >
 <author>Tomeu Borras Riera</author>
 <class>PedidosProveedorListBase</class>
 <widget class="QWidget" name="PedidosProveedorListBase" >
  <property name="enabled" >
   <bool>true</bool>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Pedidos a proveedores</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../bulmages.qrc" >:/Bulmafact48x48/images/png/48x48/i_bulmages_compras_listado.48x48.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>5</number>
   </property>
   <property name="spacing" >
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label1" >
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../../bulmages.qrc" >:/Bulmafact48x48/images/png/48x48/i_bulmages_compras_pedidos_listado.48x48.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="m_botonera" >
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>5</number>
        </property>
        <property name="spacing" >
         <number>5</number>
        </property>
        <item>
         <widget class="QToolButton" name="mui_crear" >
          <property name="toolTip" >
           <string>Nuevo pedido proveedor</string>
          </property>
          <property name="whatsThis" >
           <string>Nuevo pedido proveedor</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages.qrc" >:/Genericos32x32/images/png/i_fileNew.xpm</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_editar" >
          <property name="toolTip" >
           <string>Editar pedido proveedor</string>
          </property>
          <property name="whatsThis" >
           <string>Editar pedido proveedor</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages.qrc" >:/Genericos32x32/images/png/i_edit.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_borrar" >
          <property name="toolTip" >
           <string>Borrar pedido proveedor</string>
          </property>
          <property name="whatsThis" >
           <string>Borrar pedido proveedor</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages.qrc" >:/Genericos32x32/images/png/i_editDelete.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_imprimir" >
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Imprimir listado</string>
          </property>
          <property name="whatsThis" >
           <string>Imprimir listado</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages.qrc" >:/Genericos32x32/images/png/i_filePrint.xpm</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_filtrar" >
          <property name="toolTip" >
           <string>Filtrar pedidos</string>
          </property>
          <property name="whatsThis" >
           <string>Filtrar pedidos</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages.qrc" >:/Genericos32x32/images/png/i_find.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_configurar" >
          <property name="toolTip" >
           <string>Configurar listado</string>
          </property>
          <property name="whatsThis" >
           <string>Configurar listado</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages.qrc" >:/Genericos32x32/images/png/i_packageSettings.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_actualizar" >
          <property name="toolTip" >
           <string>Actualizar listado</string>
          </property>
          <property name="whatsThis" >
           <string>Actualizar listado</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages.qrc" >:/Genericos32x32/images/png/i_reload.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>15</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label2" >
          <property name="text" >
           <string>Buscar:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_filtro" />
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="m_busqueda" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" colspan="2" >
       <widget class="QCheckBox" name="m_procesados" >
        <property name="text" >
         <string>Solo procesados</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label6" >
          <property name="text" >
           <string>Fecha final:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="BusquedaFecha" native="1" name="m_fechafin" />
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label7" >
          <property name="text" >
           <string>Fecha inicial:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="BusquedaFecha" native="1" name="m_fechain" />
        </item>
       </layout>
      </item>
      <item row="2" column="0" >
       <widget class="BusquedaProveedor" native="1" name="m_proveedor" >
        <property name="minimumSize" >
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="BusquedaArticulo" native="1" name="m_articulo" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="PedidosProveedorListSubform" name="mui_list" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>211</width>
         <height>37</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label3" >
       <property name="text" >
        <string>Suma de totales de pedidos:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_total" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="font" >
        <font>
         <family>Sans Serif</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text" >
        <string>0.00</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton" >
       <property name="text" >
        <string>&amp;Cerrar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="2" margin="5" />
 <customwidgets>
  <customwidget>
   <class>BusquedaArticulo</class>
   <extends>QWidget</extends>
   <header>busquedaarticulo.h</header>
  </customwidget>
  <customwidget>
   <class>BusquedaProveedor</class>
   <extends>QWidget</extends>
   <header>busquedaproveedor.h</header>
  </customwidget>
  <customwidget>
   <class>BusquedaFecha</class>
   <extends>QWidget</extends>
   <header>../../../bulmalib/busquedafecha.h</header>
  </customwidget>
  <customwidget>
   <class>PedidosProveedorListSubform</class>
   <extends>QTableWidget</extends>
   <header>pedidosproveedorlist.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../bulmages.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>mui_filtrar</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_busqueda</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>231</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel" >
     <x>227</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
